src/Raven.Server/Documents/Replication/OutgoingReplicationHandler.cs

```diff
@@ -475,6 +479,8 @@ private void Replicate()
             }
         }
 
+        public long RaftIndex;
```

Comment:
I don't understand what is the purpose of this. 
This just tells you what is the last raft index that we sent this with, not whatever we properly updated anything internally about it.

Note that the `LastRaftCommitIndex` is updated *without* waiting for the commands to actually finish applying (async)